### ISSUE TITLE
[F] Tray icon to show active Note folder name

### ISSUE THREAD

**Issue by truthsword** [pbek/QOwnNotes#1451]
#### Expected behavior

Several possibilities...
a. Hovering over the tray icon gives popup identifying the active note folder
b. Right-clicking tray icon displays active not folder name (optionally, a flyout menu on right-click provides a means to change the active note folder)
c. ???

#### Actual behavior

Feature suggestion

#### Relevant log output in the Log panel

<details><summary>Expand</summary>

```

```
</details>

#### Information about the application, settings and environment

<details><summary>Expand</summary>

QOwnNotes Debug Information
===========================

## General Info

**Current Date**: `Mon Nov 25 10:58:56 2019`
**Version**: `19.11.20`
**Build date**: `Nov 23 2019`
**Build number**: `4938`
**Platform**: `windows`
**Operating System**: `Windows 10 (10.0)`
**Build architecture**: `i386`
**Current architecture**: `x86_64`
**Release**: `AppVeyor`
**Qt Version (build)**: `5.13.2`
**Qt Version (runtime)**: `5.13.2`
**Portable mode**: `no`
</details>

**Comment by pbek** [comment]
Thank you for your suggestion.

**Comment by pbek** [comment]
Trickier than I thought, will be as good as it gets.

**Comment by pbek** [comment]
Showing the note folder in the tray menu directly as label didn't work for me.

## 19.11.23
- you now change the current note folder in the tray icon
    - the current note folder is marked with an icon (making it bold didn't seem to work)

**Comment by pbek** [comment]
There now is a new release, could you please test it and report if it works for you?

I don't know if that's a KDE bug but currently I have to restart Plasmashell to see that a note folder actually was changed in the tray menu.

**Comment by truthsword** [comment]
This is quite nice. Thank you.

A small thing I noticed, is when a select a note folder, the context menu (from right-clicking the tray icon) does not auto-close (disappear) until I click elsewhere on the screen (Win10 x64).

![](https://i.imgur.com/wffdw2G.png_)

**Comment by pbek** [comment]
Must be a Qt Windows thing. It closes on KDE Neon. Does it close if you click other menu items?

**Comment by truthsword** [comment]
There are several things odd with the tray launcher (I've never used some of these features before).

1.  When I select a recent note, and the program is minimized, it opens the program window but does not display the note I selected. However, the tray context menu goes away.

1.  When I select a recent note, and the program window is **open**, the program window does not display the note I selected, and the tray context menu remains as appears when initially right-clicked.

1.  When I change the active folder, and the program is minimized, the program remains minimized, and the tray context menu remains as appears when right-clicked.

1.  When I change the active folder, and the program window is **open**, the program window changes the active folder I selected, and the tray context menu remains as appears when right-clicked.
